Source/ACE.Server/Managers/TransferManager.cs

```diff
@@ -0,0 +1,788 @@
+using System;
```

Comment:
I *THINK* I'd prefer this file in the following namespace instead:

Source/ACE.Server/TransferServer/

Reply:
I don't.  The server is just a server, but the transfer manager is an ACE feature, not a server feature.  I know you don't like having an HTTP server running in ACE but that doesn't mean features dependent on the HTTP server are the HTTP server.  This is especially the case if we tie TransferManager into the AC server as well, or instead of, you wouldn't want TransferManager to be moved into `ACE.Networking`.